File: dart/dsa/linkedlist/linkedlist.dart
class Node {
  int data;
  Node? next;
  Node(this.data);
}

void traversal(Node? head) {
  while (head != null) {
    print(head.data);
    head = head.next;
  }
}

void main() {
  Node head = Node(8);

  Node second = Node(8);

  Node third = Node(8);
  head.next = second;
  second.next = third;
  third.next = null;
  traversal(head);
}

File: python/dsa/linkedlist/linkedlist.py
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

    def traversal(head):
        while head:
            print(f'{head.data}')
            head = head.next


head = Node(9)
second = Node(7)
third = Node(0)

head.next  = second
second.next = third

Node.traversal(head)